### PR TITLE
[12.0] Add project tags on search view

### PR DESCRIPTION
**Pull request by ovnicraft**

### CODE REVIEW

**Comment by ovnicraft** [comment]
Tested it on runbot, works fine.

**Review by moylop260 — APPROVED** [review]

File: project_tag/views/project.xml [hunk 1]
```diff
@@ -58,4 +58,19 @@
         </xpath>
     </field>
   </record>
+
+
+<!-- Search view -->
```
Comment (astirpe) [inline project_tag/views/project.xml]:
```suggestion
 <!-- Search view -->
```

Fix indentation

File: project_tag/views/project.xml [hunk 2]
```diff
@@ -58,4 +58,19 @@
         </xpath>
     </field>
   </record>
+
+
+<!-- Search view -->
+  <record id="project_tags_view_inherit_search" model="ir.ui.view">
```
Comment (astirpe) [inline project_tag/views/project.xml]:
```suggestion
  <record id="view_project_project_filter" model="ir.ui.view">
```

as per [OCA guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#inherited-xml)

File: project_tag/views/project.xml [hunk 3]
```diff
@@ -58,4 +58,19 @@
         </xpath>
     </field>
   </record>
+
+
```
Comment (astirpe) [inline project_tag/views/project.xml]:
```suggestion
```

(too many empty lines)

**Review by astirpe — CHANGES_REQUESTED** [review]
Can you increase version in the manifest?

File: project_tag/views/project.xml (hunk 1 above)
Comment (astirpe) [reply project_tag/views/project.xml]: My mistake: there were here two missing blank characters, I added only one so now is still missing the other one  :see_no_evil: :smile:

**Review by astirpe — APPROVED** [review]

**Review by patrickrwilson — APPROVED** [review]

**Pull request merged**